robot/Cumulus/tests/browser/levels/create_edit_delete_level.robot

```diff
@@ -8,115 +8,99 @@ Library         cumulusci.robotframework.PageObjects
+    ...                                                 Maximum Amount=0.99
+    Enter Level Dd Values
+    ...                                                 Source Field=Smallest Gift
+
+    Click Button                                        Save
+    Go To Page                                          Details
+    ...                                                 Level__c
+    ...                                                 object_id=${level_id}
+
     Wait Until Loading Is Complete
-    Navigate To And Validate Field Value          Smallest Gift    contains    $0.75    section=Donation Totals
-    # --------------------------------
-    # Open NPSP Settings and run the Levels batch job
-    # --------------------------------
-    Open NPSP Settings      Bulk Data Processes         Level Assignment Batch
-    Click Settings Button    idPanelLvlAssignBatch    Run Batch
-    Wait For Batch To Process        LVL_LevelAssign_BATCH    Completed
-    # --------------------------------
-    # Return to the Contact to validate the updated Level field
-    # --------------------------------
-    Go To Page    Details    Contact       object_id=${contact_id}
-    Navigate To And Validate Field Value          Level    contains    ${level_name}    section=Donation Totals
 
+    Navigate To And Validate Field Value    Maximum Amount (<)     contains       0.99
+    Navigate To And Validate Field Value    Source Field           contains       npo02__SmallestAmount__c
+
+
+2 Validate Level Assignment in Batch Job
+    [Documentation]                      Create a contact, edit the smallgift field value to apply a valid
+    ...                                  level by running the batch process
+
+    [tags]                                  W-038641                 feature:Level
     # --------------------------------
-    # Modify the SmallestGift field to change the applied level
+    # Modify the SmallestGift field to allow the level to be applied
     # --------------------------------
-    Scroll Element Into View    text:Donation Totals
-    Click Button       title:Edit Smallest Gift
-    Wait For Locator  record.footer
-    Populate Field          Smallest Gift     2.0
-    Click Record Button     Save
-    Wait Until Loading Is Complete
-    Navigate To And Validate Field Value           Smallest Gift    contains    $2.00    section=Donation Totals
+    Setupdata                               contact                   contact_data=${contact_fields}
+    Set Global Variable                     ${data}
+    Salesforce Update                       Contact                   ${data}[contact][Id]  npo02__SmallestAmount__c=0.75
+    Go To Page                              Details
+    ...                                     Contact
+    ...                                     object_id=${data}[contact][Id]
+
+    Navigate To And Validate Field Value    Smallest Gift    contains    $0.75
```

Comment:
@rjanjanam did we want to hardcode $0.75 here?

Reply:
As per the testcase that is the value we want to compare. But removed hardcoding and passing variables